### PR TITLE
[181] How's the weather?

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[181] How's the weather?`
- `Weather php`

### CODE REVIEW

**Review by adaugochi — CHANGES_REQUESTED** [review]
You have some node module file that is pushed along with this PR. kindly delete them. Click https://github.com/Dufuna-CodeCamp/Dufuna-CodeCamp20/pull/1075/files to see what I am talking about

**Comment by kazeemerinfolami** [comment]
Deleted 🚮 , thank you.

File: submissions/kazeemErinfolami/PHP/weather.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+function temperature ($celsius){
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/PHP/weather.php:2]: Kindly format this block of code to make it lool readable

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by kazeemerinfolami** [comment]
Corrected 👍🏿

**Review by adaugochi — APPROVED** [review]

**Pull request merged**